test-cases/microbenchmarking/amazon_perf_simple_query_i4i.yaml

```diff
@@ -0,0 +1,18 @@
+test_duration: 60
+backtrace_decoding: false
+cluster_backend: 'aws'
+
+n_loaders: 0
+n_monitor_nodes: 0
+n_db_nodes: 1
+use_mgmt: false
+
+instance_type_db: 'i4i.2xlarge'
```

Comment:
Why i4i is chosen?
Why not the equivalent of the one that was chosen to run for arm?

Reply:
agree will set c6i.large

Reply:
fixed